src/estimators/two_view_geometry.cc

```diff
@@ -361,6 +361,10 @@ void TwoViewGeometry::EstimateCalibrated(
                         num_inliers, *best_inlier_mask, options)) {
       config = ConfigurationType::WATERMARK;
     }
+
+    if (options.compute_relative_pose) {
+      EstimateRelativePose(camera1, points1, camera2, points2);
```

Comment:
it seems like we could also add this to EstimateUncalibrated/EstimateHomography?

Reply:
Sure, added.